dnf5/commands/repoquery/repoquery.cpp

```diff
@@ -150,6 +227,22 @@ void RepoqueryCommand::load_additional_packages() {
     }
 }
 
+// In case of input being nevras -> resolve them to packages
+static libdnf::rpm::PackageQuery resolve_nevras_to_packges(
```

Comment:
Do we need `PackageQuery`? Or is `PackageSet` enough?
When I look below, the result is used as a `PackageSet`.

Maybe we can use `PackageSet`:
```
static libdnf::rpm::PackageSet resolve_nevras_to_packges(
    libdnf::Base & base, const std::vector<std::string> & nevra_globs, const libdnf::rpm::PackageQuery & base_query) {
    auto resolved_nevras = libdnf::rpm::PackageSet(base);
    ...
```